Add link to discord community

gsaslis opened the pull request:
Note to reviewers: 
the discord icon is not showing because it was only added to font-awesome @ 5.0.0

Upgrading font-awesome is something I don't feel comfortable doing, primarily because I don't have a fully working local env to test out the changes to the website.  (https://github.com/softwarecrafters/website#-contributing-to-the-website is still "TBD")

gsaslis committed:
Adds link to discord community

the icon is not showing because it was only added to font-awesome @ 5.0.0

Upgrading font-awesome is something I don't feel comfortable doing, primarily because I don't have a fully working local env to test out the changes to the website.  (https://github.com/softwarecrafters/website#-contributing-to-the-website is still "TBD")

gsaslis commented:
ah, cool, I just noticed there's a deploy preview here ! nicely done!!

gsaslis commented:
I just noticed the discord invite is personalized ("yorgos invites you") and I was not aiming for that. I'll look into whether a vanilla one is possible.

gsaslis commented:
I wasn't able to find an easy way of the personalized invite: https://support.discord.com/hc/en-us/articles/208866998-Invites-101

I don't mind if we keep that - I just don't like that it is so person-centric... 

Thoughts / ideas are more than welcome!

rradczewski commented:
Regarding font awesome: please use an image for the logo, i won't have time soon to upgrade font awesome, i vaguely remember the license changing so it might not be trivial.

gsaslis committed:
Moves to discord icon from font-awesome icon

discord icon is only available on font-awesome 5.0.0+

gsaslis commented:
@rradczewski should be all set now

gsaslis committed:
updates discord invite link

this one will only grant temporary access. If folks don't accept the code of conduct (which happens through a discord role assignment, powered by carl-bot), they won't stay on the server.
